### PR TITLE
chore: [CDS-71738]: remove CDS_TAS_NG FF

### PR DESCRIPTION
**Pull request by PratyushGar**
### Summary

- Remove the CDS_TAS_NG.
- TAS is now behind FF - NG_SVC_ENV_REDESIGN

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by PratyushGar** [comment]
image
Coverage is failing for existing files

**Review by rameshx — DISMISSED** [review]

**Review by srdjan-harness — DISMISSED** [review]

**Review by vikrant-harness — APPROVED** [review]

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceManifestOverride/ServiceManifestOverride.tsx [hunk 1]
```diff
@@ -417,9 +417,9 @@ function ServiceManifestOverride({
       return serviceType === ServiceDeploymentType.TAS ? TASOverrideManifests : AllowedManifestOverrideTypes
     } else {
       // Environment Configurations
-      return getAllowedOverrideManifests({ CDS_TAS_NG })
+      return getAllowedOverrideManifests({ NG_SVC_ENV_REDESIGN })
```
Comment (pankajsingh-harness) [inline src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceManifestOverride/ServiceManifestOverride.tsx:420]: Is this change in FF expected?

**Review by ishantmehtaharness — DISMISSED** [review]
Looks Good

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceManifestOverride/ServiceManifestOverride.tsx (hunk 1 above)
Comment (PratyushGar) [reply src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceManifestOverride/ServiceManifestOverride.tsx:420]: Yes, the previous idea for allowedOverrideManifest was to have a list dependent on deployment type, now `TAS_NG` has been merged with the `NG_SVC_ENV_REDESIGN`, still other new deployment types can add it behind FF.

**Review by sunnykesh-harness — APPROVED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

**Review by vikrant-harness — APPROVED** [review]

**Review by sunnykesh-harness — APPROVED** [review]

**Comment by PratyushGar** [comment]
Coverage is failing due to existing generic files
image

**Pull request merged**